app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/upload/package.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers
+
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.upscan.UploadReference
+
+package object upload {
+
+  implicit class UploadOps[A](val maybeAnswer: Option[A]) extends AnyVal {
```

Comment:
rename this class to `FileUploadOps` when you have finished renaming the package to the `fileUpload`